doc/examples.rst

```diff
@@ -152,3 +152,123 @@ Instantiated ``init.lua`` content:
 
     require("fiber").sleep(1)
 
+Packing environments
+----------------------------------
+
+For example, we want to pack a single application. Here is the content of the sample application::
+      single_environment/
+      ├── tarantool.yaml
+      └── init.lua
+
+``tarantool.yaml``:
+
+.. code-block:: yaml
+
+    tt:
+        app:
+
+For packing it into tarball, call:
+
+.. code-block:: bash
+
+   $ tt pack tgz
+      • Apps to pack: single_environment
+      • Generating new tarantool.yaml for the new package
+      • The passed bundle packed into temporary directory: /var/folders/T/tt_pack1901831662
```

Comment:
You have changed log level of this message. Looks like, it should not be printed bye default. Please, update examples.

Reply:
Fixed, thank you.